boards/m5stack/m5stack_atoms3_lite/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: m5stack_atoms3_lite
```

Comment:
this board should just be a variant of the original since it's the same but without a display or whatnot

Reply:
This would also require updating documentation to join this board and its variant details? Or it can be still split?

Reply:
Ok, just add an index.rst and add documentation files accordingly.

Reply:
Ok, I'd rather not perform this change right now. Board is in place and will required more work. I'll check other pending issues so we can get everything in place.